recipes/xr_fresh/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xr_fresh" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mmann1123/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 000e241ca2a7ffa0f39f52aee53d4bf207cab04d8daa314a3842c661c2c51fd1
+
+build:
+  skip: true  # [py<37 or py>311]
+  skip: true  # [win]
+ 
+  script: |
+    {{ PYTHON }} -m pip install . -vv
+
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Please correct me if I'm wrong but a quick look at the `setup.py` on Github doesn't indicate you need compilers to build this. I believe it can be noarch.